paper-radio-group.html

```diff
@@ -157,34 +156,32 @@
       this.fire('paper-radio-group-changed');
     },
 
-    /**
-     * Selects the previous item. If the previous item is disabled, then it is
-     * skipped, and its previous item is selected
-     */
-    selectPrevious: function() {
-      var length = this.items.length;
-      var newIndex = Number(this._valueToIndex(this.selected));
-
-      do {
-        newIndex = (newIndex - 1 + length) % length;
-      } while (this.items[newIndex].disabled)
-
-      this._itemActivate(this._indexToValue(newIndex), this.items[newIndex]);
+    _activateFocusedItem: function() {
+      this._itemActivate(this._valueForItem(this.focusedItem), this.focusedItem);
     },
 
-    /**
-     * Selects the next item. If the next item is disabled, then it is
-     * skipped, and the next item after it is selected.
-     */
-    selectNext: function() {
-      var length = this.items.length;
-      var newIndex = Number(this._valueToIndex(this.selected));
+    _onUpKey: function(event) {
+      this._focusPrevious();
+      this._activateFocusedItem();
+      event.detail.keyboardEvent.preventDefault();
+    },
 
-      do {
-        newIndex = (newIndex + 1 + length) % length;
-      } while (this.items[newIndex].disabled)
+    _onDownKey: function(event) {
+      this._focusNext();
+      this._activateFocusedItem();
+      event.detail.keyboardEvent.preventDefault();
+    },
 
-      this._itemActivate(this._indexToValue(newIndex), this.items[newIndex]);
+    _onLeftKey: function(event) {
+      Polymer.IronMenubarBehaviorImpl._onLeftKey.apply(this, arguments);
+      this._activateFocusedItem();
+      event.detail.keyboardEvent.preventDefault();
```

Comment:
not needed, as it's already done by the implementation of `_onLeftKey`  in `IronMenubarBehaviorImpl`